src/app/components/task/SingleChoiceTask.js

```diff
@@ -164,8 +165,10 @@ class SingleChoiceTask extends Component {
               ))}
             </RadioGroup>
 
-            <div
-              style={{ display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}
+            <Box 
+              display="flex" 
+              justifyContent="flex-start" 
```

Comment:
168:17  error  Trailing spaces not allowed  no-trailing-spaces
169:29  error  Trailing spaces not allowed  no-trailing-spaces
170:42  error  Trailing spaces not allowed  no-trailing-spaces

Reply:
^ These are still unfixed